### DISCUSSION TITLE
getting_started page needs update?

### DISCUSSION THREAD

**Discussion by Cyberavater** [rrousselGit/riverpod#546] [General]
Seems like the example code for `Flutter + flutter_hooks` from here, https://riverpod.dev/docs/getting_started doesn't work.

It's giving `'MyApp.build' ('Widget Function(BuildContext)') isn't a valid override of 'ConsumerWidget.build' ('Widget Function(BuildContext, WidgetRef)')` and `Undefined name 'ref'.`

**Comment by rrousselGit** [comment]
Indeed, the build method is missing a parameter

The doc is not up-to-date with the dev release.

**Reply by Cyberavater** [reply]
I'm very new to all of this, can you point me to an updated guide? Thanks!

**Reply by rrousselGit** [reply]
Do:

```
Widget build(BuildContext context, WidgetRef ref) {
..
}
